server_modules/scores.js

```diff
@@ -47,4 +71,46 @@ exports.route = function(app) {
         }).catch(utils.sendError(res)).done();
     });
 
+    //save a new score
+    app.post('/scores/create',function(req,res) {
+        var body = JSON.parse(req.body);
+        var scoresheet = body.scoresheet;
+        var score = body.score;
+
+        changeScores(function(result) {
+            result.scores.push(score);
+            return result;
+        })
+        .then(fileSystem.writeFile(fileSystem.getDataFilePath("scoresheets/" + score.file), req.body))
+        .then(function(scores) {
+            res.json(scores).end();
+        }).catch(utils.sendError(res));
+
+    });
+
+    //delete a score at an id
+    app.post('/scores/delete/:id',function(req,res) {
+        changeScores(function(result) {
+            var index = result.scores.findIndex((score) => score.id === req.params.id);
```

Comment:
What happens if the `id` isn't found? (e.g. conficting edit actions)
Return a more appropriate error to the client than a crash?